packages/core/deploy/036_deploy_across_ovm_bridge_deposit_box.js

```diff
@@ -0,0 +1,30 @@
+// This deploy script should be run on an optimism provider.
+const { getAddress } = require("@uma/contracts-node");
+const { ZERO_ADDRESS } = require("@uma/common");
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const chainId = await web3.eth.net.getId();
+
+  if (chainId != "69" || chainId != "10") throw new Error("This deploy only works on optimism chains");
```

Comment:
Interesting, will this break tests that run all deployments by default? Is there a way, instead to have a switch where we deploy the OVM version or the non-OVM version of the deposit contract depending on which chain rather than erroring?

Reply:
ye, this broke things. I've changed it now to ignore these migrations unless you are on those chains. Not sure what the best pattern is for this tbh.